queries/elixir.scm

```diff
@@ -0,0 +1,41 @@
+(comment) @comment
+(anonymous_function) @anonymousFunction
+(arguments) @arguments_list
```

Comment:
I don't think this is a real scope 🤔. We'd like to support it, though. But if we did, you'd probably want to be able to target it from anywhere in the function.